### ISSUE TITLE
Bug on Fan Setup Auto #2

### ISSUE THREAD

**Issue by Goonfy** [aldostools/webMAN-MOD#318]
Hey,

I found a bug on latest version from this date that when i choose the fan profile Auto #2 after i use psnpatch to disable cfw syscalls and clear history that profile doesn't work, the fans stay at the same speed always, no matter the temprature, but if i use the max speed profile it works nice even with cfw syscalls disabled.

Is that normal?

**Comment by aldostools** [comment]
No, it's not normal. Your issue may need further investigation,

What do you mean with "max speed profile"? Manual fan profile?

I suspect the issue could be in the following line (but I'm not sure due I haven't tested the scenario that you described):
https://github.com/aldostools/webMAN-MOD/blob/master/include/fancontrol.h#L63

Maybe that line should be, but need someone that test it:
	if(fan_mode == SYSCON_MODE || payload_ps3hen) lv2_poke_fan(get_fan_policy_offset, restore_get_fan_policy);

If this change works, it could fix the issue on CFW but not on HEN.

**Comment by Goonfy** [comment]
i'm referring to the first and default "Auto at" profile that you put the max degrees, that is working fine even with syscalls disabled, if someone can compile it, i don't have the ps3 sdk but in last resort i can download it and compile it

**Comment by Goonfy** [comment]
i discover another bug from the fan i think, if i disable webman when using auto at profile the fan speed increases and it stays equal all the time instead of using default syscon.

any updates on the first problem?

**Comment by Joonie86** [comment]
No further feedback
